frontend/components/surplus/SurplusAuctionTransaction.stories.js

```diff
@@ -0,0 +1,157 @@
+import { storiesOf } from '@storybook/vue';
+import faker from 'faker';
+import BigNumber from 'bignumber.js';
+import { action } from '@storybook/addon-actions';
+import SurplusAuctionTransaction from './SurplusAuctionTransaction';
+import { generateFakeSurplusAuctionTransaction } from '~/helpers/generateFakeSurplusAuction';
+
+const collectAuction = generateFakeSurplusAuctionTransaction('ready-for-collection');
+
+const common = {
+    components: { SurplusAuctionTransaction },
+    data: () => ({
+        auction: generateFakeSurplusAuctionTransaction('have-bids'),
+        network: 'mainnet',
+        tokenAddress: faker.finance.ethereumAddress(),
+
+        walletMKR: undefined,
+        allowanceMKR: undefined,
+        walletAddress: undefined,
+        isAuthorizedDAI: false,
+        isAuthorizedMKR: false,
+
+        isConnectingWallet: false,
+        isRefreshingWallet: false,
+        isSettingAllowance: false,
+        isAuthorizing: false,
+        isBidding: false,
+        isCollecting: false,
+    }),
+    methods: {
+        connect() {
+            this.isConnectingWallet = true;
+            setTimeout(() => {
+                this.walletMKR = new BigNumber(faker.finance.amount(1001, 1200));
+                this.allowanceMKR = new BigNumber(faker.finance.amount(1, 2));
+                this.walletAddress = faker.finance.ethereumAddress();
+                this.isConnectingWallet = false;
+            }, 1000);
+        },
+        disconnect() {
+            this.isConnectingWallet = true;
+            setTimeout(() => {
+                this.walletMKR = undefined;
+                this.allowanceMKR = undefined;
+                this.walletAddress = undefined;
+                this.isConnectingWallet = false;
+            }, 1000);
+        },
+        setAllowanceAmount() {
+            this.isSettingAllowance = true;
+            setTimeout(() => {
+                this.allowanceMKR = new BigNumber(faker.finance.amount(800, 1000));
+                this.isSettingAllowance = false;
+            }, 1000);
+        },
+        refresh() {
+            this.isRefreshingWallet = true;
+            setTimeout(() => {
+                this.isRefreshingWallet = false;
+            }, 1000);
+        },
+        authorizeDAI() {
+            this.isAuthorizing = true;
+            setTimeout(() => {
+                this.isAuthorizedDAI = true;
+                this.isAuthorizing = false;
+            }, 1000);
+        },
+        authorizeMKR() {
+            this.isAuthorizing = true;
+            setTimeout(() => {
+                this.isAuthorizedMKR = true;
+                this.isAuthorizing = false;
+            }, 1000);
+        },
```

Comment:
Based on the https://github.com/sidestream-tech/unified-auctions-ui/issues/346#issuecomment-1184252435, should we already remove them here?

Reply:
I just removed them while you were reviewing 😄